### ISSUE TITLE
Multiple endpoint support for Developer first approach

### ISSUE THREAD

**Issue by xlight05** [wso2/product-microgateway#288]
**Description:**
We have a requiremnt to make microgateway independeant from APIM. In order to achieve that we chose developer first approach. Swagger based approach works for our requirment however developer first approach lacks functianlity to support multiple endpoints. Multiple endpoint support is available in normal microgateway using labels. However, that functionality is not available in developer first approach.

**Comment by xlight05** [comment]
I've cameup with a sample for multiple endpoint support and you can find the compression here.
https://github.com/wso2/product-microgateway/compare/2.x...xlight05:2.x

This approach takes json file as -ms setup flag.
Example json - 
```json
[
    {
      "swaggerPath":"/home/anjana/mg-test/swaggers/swag1.json",
      "endpoint":"http://0.0.0.0:9091/ser1"
    },
    {
      "swaggerPath":"/home/anjana/mg-test/swaggers/swag2.json",
      "endpoint":"http://0.0.0.0:9092/ser2"
    },
    {
      "swaggerPath":"/home/anjana/mg-test/swaggers/swag3.json",
      "endpoint":"http://0.0.0.0:9093/ser3"
    }
]

```
Appricate your feedback on this.

**Comment by xlight05** [comment]
A temporary solution was introduced in #297.
The feature is available in https://github.com/wso2/product-microgateway/tree/feature-multi-swagger

**Comment by Rajith90** [comment]
This issue is addressed here https://github.com/wso2/product-microgateway/issues/325.
And will be available with the 3.0.0 release

**Comment by praminda** [comment]
Will be addressed by #325
